### DISCUSSION TITLE
Overlapping Glyphs

### DISCUSSION THREAD

**Discussion by messede-degod** [sourcelair/xterm.js#4607] [Q&A] [answered]
Some glyph's (the skull in my case) tend to overlap on regular text characters.

## Details
- Browser and browser version: Brave v1.56.11 (Chromium -115.0.5790.102)
- OS version: Ubuntu 22
- xterm.js version: 5.2.1

### Steps to reproduce

1.  Fire up a xterm.js terminal
2.  paste the following content in the terminal : abc💀xyz
3.  what i expect to see : 
      ![image](https://github.com/xtermjs/xterm.js/assets/59089310/1adaca6e-592b-4c54-a964-d9273c136fc1)
      (i had to add a extra space after the skull to obtain the above result)
5.  what i actually see (note the overlapping skull and 'x' )
      ![image](https://github.com/xtermjs/xterm.js/assets/59089310/87a55347-8d3a-4fb7-8845-d5741bbbd30a)

### Quick testing environment (if required)
- You can open up https://shell.segfault.net and click **new** to start a xterm.js terminal.

**Comment by Tyriar** [comment, marked as answer]
This is related to the unicode version, it might be fixed by using the unicode 11 addon https://github.com/xtermjs/xterm.js/tree/master/addons/xterm-addon-unicode11. If not, it will require a later version of the unicode addon to be created

**Reply by messede-degod** [reply]
thank you for pointing this out, ill use the addon and let you know how how it pans out

**Reply by messede-degod** [reply]
![image](https://github.com/xtermjs/xterm.js/assets/59089310/1977aa62-ba93-45ce-96a2-693166619abc)
Yup, like you predicted unicode 11 fixes the issue. Now i just have have to wait for it to leave the experimental phase. 🙃

**Reply by jerch** [reply]
> ... to wait for it to leave the experimental phase.

Eww, its been in that state for like 3ys :smile_cat:. Well, it is still unstable ground, which hopefully #4519 can settle soon.
